spyder/plugins/updatemanager/confpage.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © Spyder Project Contributors
+# Licensed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+
+"""
+General entry in Preferences.
+
+For historical reasons (dating back to Spyder 2) the main class here is called
+`MainConfigPage` and its associated entry in our config system is called
+`main`.
+"""
+
+from qtpy.QtWidgets import QGroupBox, QVBoxLayout
+
+from spyder.config.base import _
+from spyder.api.preferences import PluginConfigPage
+
+
+class UpdateManagerConfigPage(PluginConfigPage):
```

Comment:
@ccordoba12, thanks for the quick suggestion. Note that
there will be 2 checkboxes: one for check at startup and one for allowing non-stable releases.

I agree that a dedicated configuration page for just 2 checkboxes seems sparse, but I decided this would be best in the interest of making the Update Manager plugin self-contained. However, I can keep the checkboxes in the Application config page if there is agreement to do so.

Reply:
We can discuss about that in our meeting tomorrow, but I'd be in favor of leaving these options in the Application config page.

Reply:
Just in case, for the preferences page to not look so empty we could also add some way to show the current Spyder version and also a way to trigger from the preferences a check for updates?

Reply:
Those are interesting ideas. Perhaps we should discuss at the next dev meeting.

Reply:
I'd prefer to move back these options to the Application config page. We can see later if there's a need to create a new config page, but for now I think it's better to leave them where they are.

Reply:
Should the configuration options also be returned to the main section or should we keep the update_manager config section?